### PR TITLE
Fix swap creation issue

### PR DESCRIPTION
**Pull request by olexiyb**
wrap swap creation with transactional-update --continue shell to fix the issue of missing swap after node reboot fix for https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/issues/1399

I see the proper transaction with 4Gb used for swap

```
# snapper list
 # │ Type   │ Pre # │ Date                     │ User │ Used Space │ Cleanup │ Description           │ Userdata
───┼────────┼───────┼──────────────────────────┼──────┼────────────┼─────────┼───────────────────────┼─────────
0  │ single │       │                          │ root │            │         │ current               │
1  │ single │       │ Sat Jun 29 19:42:50 2024 │ root │  80.40 MiB │         │ first root filesystem │
2  │ single │       │ Tue Jul  2 06:03:26 2024 │ root │  14.76 MiB │ number  │ Snapshot Update of #1 │
3- │ single │       │ Tue Jul  2 06:04:01 2024 │ root │ 352.00 KiB │ number  │ Snapshot Update of #2 │
4+ │ single │       │ Tue Jul  9 07:04:59 2024 │ root │   4.00 GiB │ number  │ Snapshot Update of #3 │
```

More info about swap creation in micros

[Creating_a_btrfs_swapfile](https://en.opensuse.org/SDB:Partitioning#Creating_a_btrfs_swapfile) and [openSUSE:Snapper_Tutorial#Swapfile](https://en.opensuse.org/openSUSE:Snapper_Tutorial#Swapfile)

Due  to [issue](https://btrfs.readthedocs.io/en/latest/Swapfile.html) we need to somehow run `swapon -a` after reboot
to solve
```
swapon: /var/lib/swap/swapfile: swapon failed: Read-only file system
```
I use systemd script

### CODE REVIEW

File: modules/host/templates/cloudinit.yaml.tpl [hunk 1]
```diff
@@ -24,10 +24,34 @@ runcmd:
 
 ${cloudinit_runcmd_common}
 
+# See https://en.opensuse.org/SDB:Partitioning#Creating_a_btrfs_swapfile
+# And https://en.opensuse.org/openSUSE:Snapper_Tutorial#Swapfile
+# according to https://btrfs.readthedocs.io/en/latest/Swapfile.html we need to somehow run swapon -a after reboot
+# Due to some `swapon: /var/lib/swap/swapfile: swapon failed: Read-only file system`
+# I use separate systemd script
```
Comment (mysticaltech) [inline modules/host/templates/cloudinit.yaml.tpl:31]: If you can move those commants to the PR itself it woudl be great please. Best to not have comments in cloudinit. If you want to point to the permalink of your PR comment with this info in code comment, that would be best to satisfy the curiosity of some.

**Review by mysticaltech — CHANGES_REQUESTED** [review]
@olexiyb Looking good, just one tiny change, and please also confirm that this is well tested and the swap on activation service works.

File: modules/host/templates/cloudinit.yaml.tpl (hunk 1 above)
Comment (olexiyb) [reply modules/host/templates/cloudinit.yaml.tpl]: emoved comments. Yes I currently running 3 clusters, each has 3 control panels with swap
Here is some captured data from one of my nodes

```
scmain-s1-zdx:/ # journalctl | grep swapon
Jul 15 06:33:05 scmain-s1-zdx cloud-init[1094]: Created symlink /etc/systemd/system/default.target.wants/swapon-late.service → /etc/systemd/system/swapon-late.service.
Jul 15 06:49:08 scmain-s1-zdx swapon[617]: swapon: /var/lib/swap/swapfile: swapon failed: Read-only file system
Jul 15 06:49:24 scmain-s1-zdx systemd[1]: swapon-late.service: Deactivated successfully.
Jul 25 01:16:12 scmain-s1-zdx swapon[616]: swapon: /var/lib/swap/swapfile: swapon failed: Read-only file system
Jul 25 01:16:32 scmain-s1-zdx systemd[1]: swapon-late.service: Deactivated successfully.
scmain-s1-zdx:/ # swapon
NAME                   TYPE SIZE USED PRIO
/var/lib/swap/swapfile file   8G 2.5M   -2
scmain-s1-zdx:/ # free -h
               total        used        free      shared  buff/cache   available
Mem:           3.7Gi       1.8Gi        82Mi        18Mi       2.0Gi       1.9Gi
Swap:          8.0Gi       2.0Mi       8.0Gi
```

Without swap I had to buy bigger node for control plan to run it stable. I have in total 3 clusters with 3 control nodes. So I am saving around 2.8 euros on each node.

**Review by mysticaltech — APPROVED** [review]
Awesome! Thanks @olexiyb

**Pull request merged**